models/customer-group/CHANGELOG.md

```diff
@@ -0,0 +1,70 @@
+# @commercetools-test-data/zone
```

Comment:
Should be empty and `customer-group` I think

Reply:
Copy pasta strikes again 🙃